VERSION

```diff
@@ -1 +1 @@
-0.12.1
+0.12.2
```

Comment:
IIRC we had something like `0.13.0-dev` on master last time I merged back. Not sure if we "enforce" this somehow?!

Reply:
:/ hm we don't have a strict process for this. master currently says `0.12.1` so this should be good to merge now. We can make a follow up PR to bump master to 0.13.0-dev and we should maybe put this in the release docs. WDYT?

Reply:
yes please. 


We removed this because `promu` was handling this, but not anyone ):

Reply:
let's fix in separate PR with release doc fix :+1: